pkg/stdcopy/stdcopy_test.go

```diff
@@ -85,6 +85,33 @@ func TestWriteDoesNotReturnNegativeWrittenBytes(t *testing.T) {
 	}
 }
 
+func multiplyString(originalString string, times int) string {
+	var buffer bytes.Buffer
+	for i := 0; i < times; i++ {
+		buffer.WriteString(originalString)
+	}
+	return buffer.String()
+}
+
+func TestStdCopyWriteAndRead(t *testing.T) {
+	buffer := new(bytes.Buffer)
+	moreThanBufLen := 33 * 1024 // enough to have to extend the buffer in StdCopy
```

Comment:
@sergeyevstifeev would prefer if you could make `32` and `1024` constants in stdcopy.go, so that they can be referenced from the tests. If we were to change those values and forget to update the test, the test wouldn't effectively test anything.